packages/junokit/src/builtin-tasks/run.ts

```diff
@@ -46,6 +47,10 @@ async function executeRunTask (
 ): Promise<any> {
   const logDebugTag = "junokit:tasks:run";
 
+  // build the ts scripts/test to js first
+  // path dependent, might find a better soln later
+  await buildTsScripts();
```

Comment:
This will be really slow if there are multiple scripts and user is trying to run only one script.

Reply:
One way would be run `yarn run build:watch`

Reply:
I think we should include it in a tutorial